.github/workflows/acceptance-tests.yaml

```diff
@@ -42,7 +42,7 @@ jobs:
           go-version-file: 'go.mod'
       - uses: hashicorp/setup-terraform@v3
         with:
-          terraform_version: '1.5.*'
+          terraform_version: '1.11.*'
```

Comment:
Looks like CI won't use this new version, even though viewing the workflow file says it should be 1.11, it keeps installing 1.5. Opened a separate PR so it's in main, assuming that'll fix it: https://github.com/PrefectHQ/terraform-provider-prefect/pull/404